mod/01_rating/spec/set/right/downvotee_search_spec.rb

```diff
@@ -24,7 +24,7 @@
           vcc2.save!
         end
         metric_downvotee_search_card = Card.fetch "#{apple.name}+metric+downvotee search"
-        result = metric_downvotee_search_card.item_cards
+        result = metric_downvotee_search_card.format.get_search_result
```

Comment:
`item_cards` is do the raw query which won't follow the order of votes. 
```
def item_names params={}
  Card.search(query(params.merge(:return=>:name)))
end
```

`get_search_result` will re-order according the users' voting history. 

view `card_list` will render the result using `search_results` which will include the re-ordering.